src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/AccessTokenHandler.cs

```diff
@@ -0,0 +1,145 @@
+using Altinn.Common.AccessToken.Configuration;
+using Altinn.Common.AccessToken.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Altinn.Common.AccessToken
+{
+    public class AccessTokenHandler : AuthorizationHandler<AccessTokenRequirement>
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly ILogger _logger;
+        private readonly AccessTokenSettings _accessTokenSettings;
+        private readonly ISigningKeyResolver _signingKeysRetriever;
+
+        public AccessTokenHandler(IHttpContextAccessor httpContextAccessor,
+            ILogger<AccessTokenHandler> logger, IOptions<AccessTokenSettings> accessTokenSettings, ISigningKeyResolver signingKeysRetriever)
+        {
+                _httpContextAccessor = httpContextAccessor;
+                _logger = logger;
+                _accessTokenSettings = accessTokenSettings.Value;
+                _signingKeysRetriever = signingKeysRetriever;
+        }
+
+        /// <summary>
+        /// Handles verification of AccessTokens. Enabled with Policy on API controllers 
+        /// </summary>
+        /// <param name="context">The context</param>
+        /// <param name="requirement">The requirement for the given operation</param>
+        /// <returns></returns>
+        protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context, AccessTokenRequirement requirement)
+        {
+            StringValues tokens = GetAccessTokens();
+            if (tokens.Count != 1 & _accessTokenSettings.DisableAccesTokenVerification)
```

Comment:
Using bitwise AND instead og logical AND.

Reply:
Fixed

Reply:
Solved